mesonbuild/interpreter.py

```diff
@@ -3646,31 +3646,32 @@ def func_configure_file(self, node, args, kwargs):
         if output_format not in ('c', 'nasm'):
             raise InterpreterException('"format" possible values are "c" or "nasm".')
 
+        srcdir = self.environment.get_source_dir()
+        builddir = self.environment.get_build_dir()
+
         # Validate input
-        inputfile = None
+        inputfiles = []
         ifile_abs = None
         if 'input' in kwargs:
-            inputfile = kwargs['input']
-            if isinstance(inputfile, list):
-                if len(inputfile) != 1:
-                    m = "Keyword argument 'input' requires exactly one file"
-                    raise InterpreterException(m)
-                inputfile = inputfile[0]
-            if not isinstance(inputfile, (str, mesonlib.File)):
-                raise InterpreterException('Input must be a string or a file')
-            if isinstance(inputfile, str):
-                inputfile = mesonlib.File.from_source_file(self.environment.source_dir,
-                                                           self.subdir, inputfile)
-            ifile_abs = inputfile.absolute_path(self.environment.source_dir,
-                                                self.environment.build_dir)
+            inputfiles = kwargs['input']
+            if not isinstance(inputfiles, list):
+                inputfiles = [inputfiles]
+            if len(inputfiles) != 1:
+               if 'command' in kwargs:
```

Comment:
__[Flake8]__ [E111] indentation is not a multiple of four ([link](https://sider.review/gh/repos/19784232/pulls/5701#issue-4392889))

<sub>posted by Sider</sub>